epub33/a11y-tech/index.html

```diff
@@ -48,6 +48,7 @@
 					repoURL: "https://github.com/w3c/epub-specs",
 					branch: "main"
 				},
+				xref: ["epub-33"],
```

Comment:
Are we sure that is enough? In the other specs I added explicitly the "web-platform" profile in case we want to use html, fetch, etc, terms as well via xref. This may be unnecessary for the a11y spec right now, but may be in future...

(Or is the web-platform a default thing for respec? In which case we may want to simplify #2245)

Reply:
We don't reference anything from those specs now and I don't expect we would in the future, at least not in this document. The specific details of how you implement the objectives are in the techniques document. This one's meant to provide higher-level requirements, like WCAG.

I didn't realize web-platform covered them all, though, so we could use that in place of the html reference I put in to future-proof it.